client/src/client.rs

```diff
@@ -1627,8 +1627,8 @@ pub mod permission {
     use super::*;
 
     /// Construct a query to get all registered [`PermissionTokenDefinition`]s
-    pub const fn all_definitions() -> FindAllPermissionTokenDefinitions {
-        FindAllPermissionTokenDefinitions {}
+    pub const fn all_definitions() -> FindPermissionTokenSchema {
```

Comment:
Hm, now `Find` here looks redundant for me. It feels like a lot of work for Iroha but in fact it's just a getter

Reply:
I mean just naming, not the whole query

Reply:
fixed